fdbclient/vexillographer/fdb.options

```diff
@@ -104,6 +104,8 @@ description is not currently required but encouraged.
     <Option name="TLS_password" code="54"
             paramType="String" paramDescription="key passphrase"
             description="Set the passphrase for encrypted private key. Password should be set before setting the key for the password to be used." />
+    <Option name="TLS_disable_plaintext_connection" code="55"
+            description="Disable non-TLS connections" />
```

Comment:
It could be more descriptive: e.g. `Prevent client from connecting to a non-TLS endpoint by throwing an error`.